.github/workflows/build.yml

```diff
@@ -63,6 +63,68 @@ jobs:
           ccache --show-stats
         env:
           CCACHE_MAXSIZE: 500M
+  linux-arm64:
+    name: linux-arm64
+    runs-on: ubuntu-24.04-arm
+    env:
+      CMAKE_OPTS: >-
+        -DUSE_WERROR=ON
+        -DCMAKE_BUILD_TYPE=RelWithDebInfo
+        -DUSE_COMPILE_CACHE=ON
+      CCACHE_MAXSIZE: 0
+      CCACHE_NOCOMPRESS: 1
+      MAKEFLAGS: -j2
+      DEBIAN_FRONTEND: noninteractive
+    steps:
+      - name: Configure git
+        run: git config --global --add safe.directory "$GITHUB_WORKSPACE"
+      - name: Check out
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+          submodules: recursive
+      - name: Install system packages
+        run: |
+          sudo apt-get install -y --no-install-recommends \
+            $(xargs < .github/workflows/deps-ubuntu-24.04-arm64-gcc.txt)
+      - name: Cache ccache data
+        uses: actions/cache@v3
+        with:
+          key: ccache-${{ github.job }}-${{ github.ref }}-${{ github.run_id }}
+          restore-keys: |
+            ccache-${{ github.job }}-${{ github.ref }}-
+            ccache-${{ github.job }}-
+          path: ~/.ccache
+      - name: Configure
+        run: |
+          ccache --zero-stats
+          cmake -S . \
+                -B build \
+                -DCMAKE_INSTALL_PREFIX=./build/install \
```

Comment:
I don't believe we removed the need for `CMAKE_INSTALL_PREFIX` starting with the new `cpack` branch, which this builds atop of.